rootfs/api/models/config.py

```diff
@@ -137,13 +111,12 @@ def save(self, **kwargs):
                         # error if unsetting non-existing key
                         if key not in data:
                             raise UnprocessableEntity('{} does not exist under {}'.format(key, attr))  # noqa
-
                         data.pop(key)
                     else:
                         data[key] = value
                 setattr(self, attr, data)
 
-            self.set_healthchecks()
+            self._migrate_legacy_healthcheck()
```

Comment:
Good thinking. My current logic is that if someone is using legacy healthchecks, they're gonna have to unset them before continuing on to set a livenessProbe healthcheck.

Shall I also add a deprecation notice here to eventually rip this out after 2.4 or something?